app/src/main/java/com/iron/espresso/presentation/home/study/list/LocationFragment.kt

```diff
@@ -23,13 +26,28 @@ class LocationFragment : BaseFragment<FragmentLocationBinding>(R.layout.fragment
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        binding.studyList.adapter = studyListAdapter
+        val category = arguments?.getString(StudyListActivity.STUDY_CATEGORY).orEmpty()
+        binding.run {
+            studyList.adapter = studyListAdapter
+            swipeRefresh.apply {
+                setOnRefreshListener {
+                    viewModel.getStudyList(category, SORT_LENGTH)
 
-        viewModel.getStudyList("android", SORT_LENGTH)
+                    this.isRefreshing = false
+                }
+            }
+        }
 
-        viewModel.studyList.observe(viewLifecycleOwner, Observer { studyList ->
-            studyListAdapter.setItemList(studyList)
-        })
+        viewModel.run {
+            getStudyList(category, SORT_LENGTH)
+
+            studyList.observe(viewLifecycleOwner, Observer { studyList ->
+                studyListAdapter.setItemList(studyList)
+                binding.emptyView.visibleIf(studyList.isNullOrEmpty())
```

Comment:
```suggestion
                binding.emptyView.isVisible = studyList.isNullOrEmpty()
```
문제는 없지만, kotlin ktx 기본 익스텐션 함수를 쓰는게 어떨까 합니다
visibleIf 이건 binding adapter 에서 쓸려고 커스텀하게 만든거여서
기본이 있다면 기본껄 써주는게 모두에게 좋겠져

Reply:
ktx 코틀린 기본 익스텐션 사용하는 거로 변경했습니다!
80471f5